vllm/entrypoints/openai/api_server.py

```diff
@@ -519,6 +519,17 @@ async def create_score_v1(request: ScoreRequest, raw_request: Request):
 }
 
 
+@router.post("/reset_prefix_cache")
```

Comment:
it's better to have a flag so that this endpoint is opt-in, users should not send this request.

we can add an env var `VLLM_DEV_MODE`, and only expose this endpoint when `VLLM_DEV_MODE` is set.

Reply:
Yeah we can just add an argument when launching an endpoint to allow this route. I supposed most production API server doesn't want to this be enabled.